fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -2134,6 +2134,10 @@ public class Config extends ConfigBase {
         "Max cache number of external table row count"})
     public static long max_external_table_row_count_cache_num = 100000;
 
+    @ConfField(description = {"每个查询 list 文件目录数。",
```

Comment:
Add more detail explain.
And I think this is only for external table query? Better rename it.

Reply:
ok